### ISSUE TITLE
Cant get sigh resign to accept profile

### ISSUE THREAD

**Issue by PitchLabsAsh** [fastlane/fastlane#4161]
### Issue Description

sigh resign and fast lane resign action wont correctly locate provisioning profiles even though they are on the relevant machine.

The profile is on the machine, as can be seen here..

sigh manage
```
[20:14:10]: Loading Provisioning profiles from ~/Library/MobileDevice/Provisioning Profiles/
20:14:15: Provisioning profiles installed

20:14:15: match AdHoc com.autologic.assistmobile
20:14:15: match AdHoc com.autologic.vinsupport.dev
20:14:15: match AppStore com.autologic.assistmobile
20:14:15: match AppStore com.autologic.vinsupport.dev
```

##### Complete output when running fastlane, including the stack trace and command used

fastlane appstore 

```
[12:35:17]: -------------------------------------------------
[12:35:17]: --- Step: Verifying required fastlane version ---
[12:35:17]: -------------------------------------------------
[12:35:17]: fastlane version valid
[12:35:17]: ------------------------------
[12:35:17]: --- Step: default_platform ---
[12:35:17]: ------------------------------
[12:35:17]: Driving the lane 'ios appstore' 🚀
[12:35:17]: -------------------
[12:35:17]: --- Step: match ---
[12:35:17]: -------------------

+-----------------------+-------------------------------------------------+
|                         Summary for match 0.4.0                         |
+-----------------------+-------------------------------------------------+
| app_identifier        | com.autologic.assistmobile                      |
| type                  | appstore                                        |
| readonly              | true                                            |
| git_url               | git@bitbucket.org:pitchlabs/vincertificates.git |
| username              | ash.t@biffbangpow.com                           |
| keychain_name         | login.keychain                                  |
| team_id               | 6BXRV5C384                                      |
| verbose               | false                                           |
| force                 | false                                           |
| shallow_clone         | false                                           |
| force_for_new_devices | false                                           |
| skip_docs             | false                                           |
+-----------------------+-------------------------------------------------+

[12:35:18]: Cloning remote git repo...
[12:35:20]: 🔓  Successfully decrypted certificates repo
[12:35:20]: Installing certificate...
[12:35:22]: Installing provisioning profile...

+----------------------+------------------------------------------+
|                 Installed Provisioning Profile                  |
+----------------------+------------------------------------------+
| App Identifier       | com.autologic.assistmobile               |
| Type                 | appstore                                 |
| UUID                 | c91ea981-4430-4a86-9a00-db81c3cad408     |
| Environment Variable | sigh_com.autologic.assistmobile_appstore |
+----------------------+------------------------------------------+

[12:35:22]: All required keys, certificates and provisioning profiles are installed 🙌
[12:35:22]: 🔒  Successfully encrypted certificates repo
[12:35:22]: Setting Provisioning Profile type to 'app-store'
[12:35:22]: --------------------
[12:35:22]: --- Step: resign ---
[12:35:22]: --------------------
[12:35:22]: Error setting value 'match AppStore com.autologic.assistmobile' for option 'provisioning_profile'
[12:35:22]: You passed invalid parameters to 'resign'.
[12:35:22]: Check out the error below and available options by running `fastlane action resign`
[12:35:22]: Variable Dump:
[12:35:22]: {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios appstore", :SIGH_PROFILE_TYPE=>"app-store"}
[12:35:22]: Couldn't find provisiong profile at path 'match AppStore com.autologic.assistmobile'

+------+-------------------------------------+-------------+
|                     fastlane summary                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | Verifying required fastlane version | 0           |
| 2    | default_platform                    | 0           |
| 3    | match                               | 4           |
| 4    | resign                              | 0           |
+------+-------------------------------------+-------------+

[12:35:22]: fastlane finished with errors

[!] Couldn't find provisiong profile at path 'match AppStore com.autologic.assistmobile'

```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:
```ruby

fastlane_version "1.81.0"

default_platform :ios

platform :ios do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
  end

  desc "Runs all the tests"
  lane :test do
    cocoapods    
    scan
  end


  desc "Submit a new beta build to crashlytics"
  lane :beta do
	match(app_identifier: "com.autologic.assistmobile", type: "adhoc", readonly: true)
    cocoapods    

	gym(
      workspace: "VinSupport.xcworkspace",
      scheme: "AssistMobile",
      export_method: 'ad-hoc',
      silent: false,
      clean: true,
      output_directory: "build",
      output_name: "AssistMobile.ipa"   
    )

	build_number = number_of_commits
	add_git_tag(
	  grouping: 'fastlane-builds',
	  prefix: 'v',
	  build_number: build_number
	)
	push_git_tags

    crashlytics(
     	api_token: "bd415bb7c0471d0afe6f3e3e6e3a7e50a2638a22",
        build_secret: "e60f39058f1ca0d30423f3bdd4954a1a61fafba3484f678734968ea3ccb4da4a",
		groups: "internal"
    )
	upload_symbols_to_crashlytics(dsym_path: "./build/AssistMobile.app.dSYM.zip")

#        notes: change_log,
#        notifications: notify_testers

  end


  desc "Deploy a new version to the App Store, run only after beta lane success"
  lane :appstore do
	match(app_identifier: "com.autologic.assistmobile", type: "appstore", readonly: true)

# try using resign action
        resign(ipa: "./build/AssistMobile.ipa",
                signing_identity: "iPhone Distribution: Autologic Diagnostics Limited (6BXRV5C384)",
                provisioning_profile: "match AppStore com.autologic.assistmobile"
        )

# try using sigh resign from shell action
#    sh "sigh resign ./build/AssistMobile.ipa --signing_identity 'iPhone Distribution: Autologic Diagnostics Limited (6BXRV5C384)' -n 'match AppStore com.autologic.assistmobile'"

  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end


```

### Environment

fastlane version 1.81.0
sigh version 1.6.1

rbenv running ruby 2.2.1

**Comment by asfalcone** [comment]
Can you please include versions of `fastlane_core` and `sigh`? It would also be super awesome if you could use the [ISSUE TEMPLATE](https://github.com/fastlane/fastlane/blob/master/ISSUE_TEMPLATE.md).

**Comment by PitchLabsAsh** [comment]
Sorry will use the template properly in future.

sigh 1.6.1
fastlane 1.81.0

**Comment by PitchLabsAsh** [comment]
Updated original comment to use issue template

**Comment by TKBurner** [comment]
@PitchLabsAsh Hi, It looks like your provisioning profile is set wrong. Are you sure you mean to set it as:

provisioning_profile: "match AppStore com.autologic.assistmobile"

**Comment by PitchLabsAsh** [comment]
If you look at the output of sign manage that is the name that match generated.

sigh manage
```
[20:14:10]: Loading Provisioning profiles from ~/Library/MobileDevice/Provisioning Profiles/
20:14:15: Provisioning profiles installed

20:14:15: match AdHoc com.autologic.assistmobile
20:14:15: match AdHoc com.autologic.vinsupport.dev
20:14:15: match AppStore com.autologic.assistmobile
20:14:15: match AppStore com.autologic.vinsupport.dev
```

**Comment by asfalcone** [comment]
I believe this is a duplicate of #3934. Going to close this one as well 👍
